src/components/helpers/html_content_helpers.ts

```diff
@@ -0,0 +1,25 @@
+import { HtmlContent } from "../composer/composer/composer";
+
+export function getHtmlContentFromPattern(
```

Comment:
I think that's worth an docstring with an example.
And maybe even a few unit tests demonstrating what it does/how it works